shared/structures/src/members/PlatformMember.ts

```diff
@@ -536,10 +541,45 @@ export class PlatformMember implements ObjectWithRecords {
         return status;
     }
 
+    get hasFutureMembership(): boolean {
+        const now = new Date();
+
+        for (const t of this.patchedMember.platformMemberships) {
+            const organization = this.organizations.find(o => o.id === t.organizationId);
+            if (!organization) {
+                continue;
+            }
+
+            if (t.startDate > now) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
     get shouldApplyReducedPrice() {
         return this.patchedMember.details.shouldApplyReducedPrice;
     }
 
+    getContinuousMembershipStatus({ start, end }: { start: Date; end: Date }): ContinuousMembershipStatus {
+        const memberships = this.patchedMember.platformMemberships.filter(t => !!this.organizations.find(o => o.id === t.organizationId));
+
+        for (const t of memberships) {
+            if (t.endDate >= end && t.startDate <= start) {
+                return ContinuousMembershipStatus.Full;
+            }
+        }
+
+        for (const t of memberships) {
+            if ((t.endDate < end && t.endDate > start) || (t.startDate < end && t.startDate > start)) {
+                return ContinuousMembershipStatus.Partial;
+            }
+        }
+
+        return ContinuousMembershipStatus.None;
```

Comment:
Moet ook `ContinuousMembershipStatus.Full` teruggeven als je twee aansluitingen hebt die na elkaar volgen zonder 'gat' in